src/test/java/org/springframework/integration/kafka/config/xml/KafkaOutboundAdapterParserTests.java

```diff
@@ -60,23 +61,27 @@
 
 	@Test
 	public void testOutboundAdapterConfiguration() {
-		KafkaProducerMessageHandler<?, ?> messageHandler
+		KafkaProducerMessageHandler<?, ?> messageHandler1
 			= this.appContext.getBean("kafkaOutboundChannelAdapter.handler", KafkaProducerMessageHandler.class);
-		assertThat(messageHandler).isNotNull();
-		assertThat(messageHandler.getOrder()).isEqualTo(3);
-		assertThat(TestUtils.getPropertyValue(messageHandler, "topicExpression.literalValue")).isEqualTo("foo");
-		assertThat(TestUtils.getPropertyValue(messageHandler, "messageKeyExpression.expression")).isEqualTo("'bar'");
-		assertThat(TestUtils.getPropertyValue(messageHandler, "partitionIdExpression.expression")).isEqualTo("'2'");
-		assertThat(TestUtils.getPropertyValue(messageHandler, "sync", Boolean.class)).isTrue();
-		assertThat(TestUtils.getPropertyValue(messageHandler, "sendTimeoutExpression.expression")).isEqualTo("1000");
-
-		messageHandler
+		assertThat(messageHandler1).isNotNull();
```

Comment:
Ah, I remembered after looking at the code. I introduced two local variables `messageHandler1` and `messageHandler2`, this way the assignments and assertions to them remain separate.